.yamllint

```diff
@@ -0,0 +1,33 @@
+rules:
+  braces: enable
+  brackets: enable
+  colons: enable
+  commas: enable
+  comments:
+    level: warning
+  comments-indentation:
+    level: warning
+  document-end: disable
+  document-start: disable
+  empty-lines: enable
+  empty-values: enable
+  hyphens: enable
+  key-duplicates: enable
+  key-ordering: disable
+  line-length:
+    level: warning
+  new-line-at-end-of-file: disable
```

Comment:
enable would be better I suppose

Reply:
Enable is not required as as it will force for `\n` character at the end of file. '\n` character is must for files used by unix tools because of POSIX standards, and I don't think that our yaml files will be handled by any unix tool. But still I'll look into it to confirm.

Reply:
In GitHub look at the end of this file. You can see a red icon with a circle and arrow. And if you mouse over, you can see a message: `No newline at end of file` I would prefer not to see a message like that.